workspaces/ui-v2/src/pages/changelogHistory/ChangelogHistory.tsx

```diff
@@ -43,20 +54,22 @@ export const ChangelogHistory: FC = () => {
                 <div className={classes.commitControls}>
                   <Button
                     className={classes.commitCompareButton}
-                    href={
-                      i === 0
-                        ? documentationPage.linkTo()
-                        : changelogPage.linkTo(batchCommit.batchId)
+                    onClick={() =>
```

Comment:
@JaapRood I know you changed this in your previous PR - realized as part of building this, we need to actually use onClick so we use the react-router redirect rather than the full page browser redirect.

From my testing, we still maintain the relevant keyboard navigation behavior which is provided by material ui (i.e. we can press enter when focused on the button)

Reply:
Lets rewrite this with a `Link` from react-router then, as no href and only an on click handler causes accessibility and ux issues, keyboard navigation aside.